Caddyfile.dev

```diff
@@ -32,5 +32,5 @@
       font-src 'self' https://fonts.googleapis.com 
         https://fonts.gstatic.com https://ka-f.fontawesome.com;
       frame-src 'self' localhost:* 
-        identity.bitclout.com identity.bitclout.blue identity.bitclout.green;"
+        identity.bitclout.com identity.bitclout.blue identity.bitclout.green www.youtube.com player.vimeo.com;"
```

Comment:
Ah, thank you for the nice clean PR. I just pushed a fix, but I wish I'd seen this before-hand. Next time.